lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -35,10 +38,10 @@ export type ClientConfiguration = {
 };
 
 /**
- * @type AuthOptions: Use this to configure the auth options in the Configuration object
+ * Use this to configure the auth options in the Configuration object
  *
- *  - clientId                    - Client ID of your app registered with our Application registration portal : https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredAppsPreview in Microsoft Identity Platform
- *  - authority                   - You can configure a specific authority, defaults to " " or "https://login.microsoftonline.com/common"
```

Comment:
Can we add comments for `knownAuthorities` and `cloudDiscoveryMetadata`?

Reply:
+1

Reply:
Yes! Added.